backend/src/service/Jimmi.class.ts

```diff
@@ -1,9 +1,241 @@
   }
+
+  /**
+   * Initialize the browser page.
+   * @returns
+   */
+  async init(debug = false): Promise<void> {
+    if (this.isInitalized) return;
+    await this.page.goto(config.browser.bridge, { waitUntil: 'load' });
+
+    if (debug) {
+      this.page
+        .on(
+          'console',
+          (msg) => console.log(`${msg.type().slice(0, 3).toUpperCase()} ${msg.text()}`),
+        )
+        .on('pageerror', ({ msg }) => console.log(msg))
+        .on('response', (res) => console.log(`${res.status()} ${res.url()}`))
+        .on('requestfailed', (req) => console.log(`${req.failure().errorText} ${req.url()}`));
+    }
+
+    this.isInitalized = true;
+  }
+
+  /**
+   * Join conference on a given instance and room.
+   * Will reset the audioplay if a song is currently bering played.
+   *
+   * @param domain - domain of jitsi instance
+   * @param roomName - name of the room
+   * @returns Jimmi instance
+   */
+  async join(domain: string, roomName: string): Promise<ThisType<Jimmi>> {
+    if (this.instance !== undefined && this.roomName) {
+      await this.page.goto(config.browser.bridge, { waitUntil: 'load' });
+    }
+
+    const { gain } = config;
+    await this.page.evaluate(
+      `joinConference('${domain}', '${roomName}', '${this.botName}', ${gain})`,
+    );
+
+    this.exposeListenerFunction(this.onAudioEnded);
+    this.exposeListenerFunction(this.participantKickedOut);
+    return this;
+  }
+
+  /** *********************
+   *       GETTERS
+   * ********************* */
+
+  /**
+   * Status overview about the instance.
+   */
+  get status() {
+    return {
+      id: this.id,
+      name: this.botName,
+      isPlaying: this.isAudioPlaying,
+      conference: this.conference,
+      currentTrack: this.currentTrack,
+      queueLength: this.queue.length,
+    };
+  }
+
+  /**
+   * Overview about music.
+   */
+  get music() {
+    let status = 'stopped';
+    if (this.currentTrack !== null) status = this.isAudioPlaying ? 'playing' : 'paused';
+
+    return {
+      status,
+      queue: this.queue,
+      current: this.currentTrack,
+    };
+  }
+
+  /**
+   * Overview about the joined conference
+   */
+  get conference() {
+    return {
+      name: this.botName,
+      instance: this.instance,
+      room: this.roomName,
+    };
+  }
+
+  /** *********************
+   *    MUSIC CONTROL
+   * ********************* */
+
+  /**
+   * Play the oldest song in the queue if available.
+   * @param url -
+   * @returns Jimmi instance
+   */
+  async play(url?: string): Promise<ThisType<Jimmi>> {
+    if (url === undefined && this.currentTrack !== null) {
+      await this.page.evaluate('void audio.play()');
+    } else if (url !== undefined) {
+      await this.page.evaluate(`playAudio('${url}')`);
+      this.currentTrack = url;
+    }
+    this.isAudioPlaying = this.currentTrack !== null;
+    return this;
+  }
+
+  /**
+   * Play the oldest song in the queue if available.
+   * @returns Jimmi instance
```

Comment:
```suggestion
   * Play the oldest song in the queue if available.
   *
   * @returns Jimmi instance
```
I hate how this looks, but I'd like to be consistent and thereby compliant to the tsdoc spec